cmd/kpod/ps.go

```diff
@@ -253,13 +284,22 @@ func getTemplateOutput(containers []*libkpod.ContainerData, opts psOptions) (psO
 		ctrID := ctr.ID
 		runningFor := units.HumanDuration(time.Since(ctr.State.Created))
 		createdAt := runningFor + " ago"
-		command := getCommand(ctr.ImageCreatedBy)
+		command := getStrFromSquareBrackets(ctr.ImageCreatedBy)
 		imageName := ctr.FromImage
 		mounts := getMounts(ctr.Mounts, opts.noTrunc)
 		ports := getPorts(ctr.Config.ExposedPorts)
 		size := units.HumanSize(float64(ctr.SizeRootFs))
 		labels := getLabels(ctr.Labels)
 
+		ctrPID := strconv.Itoa(ctr.State.Pid)
```

Comment:
Can you make this into a function that returns a structure.  So we don't repeat the code twice.

Reply:
fixed